controls/cis_sle15.yml

```diff
@@ -1051,7 +1051,14 @@ controls:
     levels:
       - l1_server
       - l1_workstation
-    status: manual # rule is missing
+    status: automated
+    notes: 
+       The audit (OVAL check) cannot be automated, 
```

Comment:
Just a side note, if you are willing to not be SCAP compliant SCE (script check engine) content could be used on this rule to automate the check.